### ISSUE TITLE
[Bookworm] Update Pensando's kernel patches for Linux 6.1

### ISSUE THREAD

**Issue by saiarcot895** [sonic-net/sonic-buildimage#16866]
#### Description

Pensando's kernel patches need to be updated for Linux 6.1

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by dgsudharsan** [comment]
@saiarcot895 Please assign to the appropriate individual

**Comment by saiarcot895** [comment]
Fixed in sonic-net/sonic-linux-kernel#353.
